stat.c

```diff
@@ -414,6 +414,18 @@ static void display_lat(const char *name, unsigned long long min,
 	free(maxp);
 }
 
+static double convert_agg_kbytes_percent(struct group_run_stats *rs, int ddir, int mean)
+{
+	double p_of_agg = 100.0;
+	if (rs->agg[ddir] < 1024) {
```

Comment:
Is this comparison the wrong way round?

Also if I've contributed nothing because my rate is so small should that count as 100% or 0%?

Reply:
I agree....missed it. Will change this in this commit